python/python/Ice/LocalExceptions.py

```diff
@@ -63,58 +61,47 @@ def operation(self):
 @final
 class ObjectNotExistException(RequestFailedException):
     """
-    This exception is raised if an object does not exist on the server, that is, if no facets with the given identity
-    exist.
+    The dispatch could not find a servant for the identity carried by the request.
     """
 
 @final
 class FacetNotExistException(RequestFailedException):
     """
-    This exception is raised if no facet with the given name exists, but at least one facet with the given identity
-    exists.
+    The dispatch could not find a servant for the identity + facet carried by the request.
     """
 
 @final
 class OperationNotExistException(RequestFailedException):
     """
-    This exception is raised if an operation for a given object does not exist on the server. Typically this is caused
-    by either the client or the server using an outdated Slice specification.
+    The dispatch could not find the operation carried by the request on the target servant. This is typically due
+    to a mismatch in the Slice definitions, such as the client using Slice definitions newer than the server's.
     """
 
 class UnknownException(LocalException):
     """
-    This exception is raised if an operation call on a server raises an unknown exception. For example, for C++, this
-    exception is raised if the server throws a C++ exception that is not directly or indirectly derived from
-    Ice::LocalException or Ice::UserException.
+    The dispatch failed with an exception that is not a LocalException or a UserException.
```

Comment:
```suggestion
    The dispatch failed with an exception that is not a LocalException or an UserException.
```

Reply:
I think `a` is correct here. User doesn't start with a vowel sound.